### PR TITLE
Implement winetricks

### PR DESCRIPTION
**Pull request by Dandraghas**
Adds an option to run winetricks with `vinegar winetricks`

this thing is extremely broken, but **at least it works™**

### CODE REVIEW

**Comment by apprehensions** [comment]
Nice Niko avatar, unfortunately:

**Comment by Thelolguy1** [comment]
I'd also like to add that winetricks can already be run externally; I don't quite see the point of having it built into vinegar.

**Comment by Dandraghas** [comment]
> I'd also like to add that winetricks can already be run externally; I don't quite see the point of having it built into vinegar.

it could run externally, but doing `vinegar winetricks` is faster than typing `WINEPREFIX=~/.local/share/vinegar/prefix WINE=/path/to/wineroot/bin/wine winetricks` especially if there is custom WineRoot set in `~/.config/vinegar/config.toml`,

also the same thing i could say about `vinegar install-webview2`, you could do it manually but it creates to much hassle

File: wine/tricks.go [hunk 1]
```diff
@@ -2,10 +2,34 @@ package wine
...
 import (
 	"log"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/vinegarhq/vinegar/internal/config"
 )
 
+func isWinerootTricksAvaliable(wineroot string) (bool, error) {
+	_, err := exec.LookPath(filepath.Join(wineroot, "bin", "winetricks"))
```
Comment (apprehensions) [inline wine/tricks.go:12]: Unnecessary, the existing `Run()` will already tell us that, and you're not wrapping the error so there is no point.

File: wine/tricks.go [hunk 2]
```diff
@@ -2,10 +2,34 @@ package wine
...
+func (p *Prefix) LaunchWinetricks(cfg *config.Config) error {
+	log.Println("Launching winetricks")
+	result, err := isWinerootTricksAvaliable(cfg.WineRoot)
+	if err != nil && !result {
+		log.Println("winetricks not found in wineroot: " + err.Error() + ", falling back to system winetricks")
+		return p.Command("winetricks").Run()
+	}
+
+	return p.Command(filepath.Join(cfg.WineRoot, "bin", "winetricks")).Run()
+}
```
Comment (apprehensions) [inline wine/tricks.go:35]:
```suggestion
func (p *Prefix) Winetricks() error {
	log.Println("Launching winetricks")
	
	return p.Command("winetricks").Run()
}
```
`winetricks` is a program. It is not part of a wine installation. and `PATH` is automatically appended to with the bin of wineroot when using the wineroot, how else would we be using the wineroot's wine?

**Review by apprehensions — CHANGES_REQUESTED** [review]

**Comment by apprehensions** [comment]
You've tested this right?

**Comment by Dandraghas** [comment]
> You've tested this right?

Yes, why ask?

**Review by apprehensions — APPROVED** [review]

**Pull request merged**